docs/bump_version.md

```diff
@@ -0,0 +1,45 @@
+# How to bump a package version for an existing recipe in ConanCenterIndex?
+
+Once a recipe is approved and merged to ConanCenterIndex, it may need to be updated with new versions released by the upstream.
+When someone needs a new package version that is not available in ConanCenterIndex, then open a new pull request adding. This is
+called a `bump version`.
+A bumping version process is when a PR only adds a new package version and nothing more. Removing older versions, or updating
+the recipe is not classified as a bump version action.
+
+## Choosing which version should be updated
+
+The first step is checking the version which should be added from the upstream. Please, avoid adding multiple versions which you do not
+need. Adding a new version will increase the building time and storage for each new package configuration.
+
+Once you detect which version should be updated, please, first check if the project `license` keeps the same.
```

Comment:
Newer versions might have more knots that Conan might need to handle (more dependencies, configuration options, etc) which might require modifying the recipe.
I think this situation should also be clarified and that due to this (as stated before) the PR won't be labeled as `bump version`.

Reply:
It's documented on https://github.com/conan-io/conan-center-index/blob/master/docs/labels.md#bump-version, but clarifying here is valid too.

Reply:
Done.